doc/whats-new-2.0.rst

```diff
@@ -3,6 +3,19 @@
 What's New In khmer 2.0?
 ########################
 
+
+Script Updates
+==============
+
+Normalize by median supports broken-paired input
```

Comment:
As long as I'm micromanaging :)

"Digital normalization script now supports mixed paired and unpaired read input." And then, below, say "(or broken-paired input)".  broken-paired is not standard bioinfo lingo.